js/foundation.reveal.js

```diff
@@ -85,7 +85,7 @@ class Reveal {
   _makeOverlay(id) {
     var $overlay = $('<div></div>')
                     .addClass('reveal-overlay')
-                    .appendTo('body');
+                    .appendTo('this.options.appendTo');
```

Comment:
In order to have the code aligned I would like to see ` .appendTo($(this.options.appendTo));